ftplugin/latex-suite/compiler.vim

```diff
@@ -591,7 +591,7 @@ function! Tex_CompileMultipleTimes()
 		" The first time we see if we need to generate the bibliography and if the .bbl file
 		" changes, we will rerun latex.
 		" We use '\\bibdata' as a check for BibTeX and '\\abx' as a check for biber.
-		if runCount == 0 && Tex_IsPresentInFile('\v\\bibdata|\\abx', mainFileName_root.'.aux')
+		if runCount == 0 && Tex_IsPresentInFile('\\bibdata|\\abx', mainFileName_root.'.aux')
```

Comment:
```\v``` is not a vertical tab but a modifier for the regular expression. How is the file called that your are editing where you need this patch to get it to work? What is the output of

    :echo Tex_GetMainFileName(':p:t:r')

when you run it in vim with the file open where you need this patch to work. Can you please verify that the filename you openend really shows this error?